### PR TITLE
Feat: support passthrough mode in colang 2

### PR DESCRIPTION
**Pull request by Pouyanpi**
To enable passthrough mode in Colang 2.x one needs to import passthrough module and activate llm continuation in the main flow.

```co

import passthrough
flow main
  activate llm continuation
  ...
  
```

- [x] Change action name and flow names
- [x] resolve flow competitions
- [x] use raw_response contextvar
- [x] add tests

### CODE REVIEW

File: nemoguardrails/colang/v2_x/library/llm.co [hunk 1]
```diff
@@ -13,6 +13,22 @@ flow bot say something like $text
   await bot say $variation
 
 
+flow bot responses to user prompt
```
Comment (schuellc-nvidia) [inline nemoguardrails/colang/v2_x/library/llm.co]: We should reuse flow `continuation on unhandled user intent` instead if possible. @Pouyanpi Let's discuss.

**Comment by schuellc-nvidia** [comment]
I am not sure whats the motivation for this feature. Could you elaborate? Thanks!

**Comment by Pouyanpi** [comment]
@schuellc-nvidia feel free to review. Is the test OK with you?.

**Review by schuellc-nvidia — APPROVED** [review]
All good

**Pull request merged**